src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,10 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer extends Bulldozer {
+    List<? extends Bulldozer> get() {
```

Comment:
This method must be overrided from MachineProducer class

Reply:
fixed